.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We don't want to check this in. Please add `.vscode` to gitignore

Reply:
I have resolved the issue.